db/seeds.rb

```diff
@@ -1,7 +1,87 @@
+<<<<<<< HEAD
```

Comment:
- [ ] This seeds.rb is full of commented code and has a syntax error in the end.  Kindly remove the unused code and delete the last line which prevents this file from running. 💪